cumulusci/utils.py

```diff
@@ -146,6 +144,17 @@ def download_extract_zip(url, target=None, subfolder=None, headers=None):
     return zip_file
 
 
+def download_extract_github(github_repo, subfolder, ref=None):
```

Comment:
@davisagli - we should consider grabbing kit's zipfile-trustworthiness-validator for both this method and download_extract_zip

and ... @wlonk - this helper is about to land in CCI core and we should probably use it in MetaDeploy!

Reply:
@cdcarter sure, but I think we can handle that separately